Documentation/DevelopmentTips.md

```diff
@@ -4,6 +4,50 @@ Tips and tricks while developing Xamarin.Android.
 
 # How do I rebuild the Mono Runtime and Native Binaries?
 
+## The short way
+
+From the top of the Xamarin Android source tree you can run make for the following
+targets, which implement the steps to rebuild the runtime and BCL described further
+down in the document:
+
+    # Show help on all the rebuild targets
+    make rebuild-help
+
+    # Rebuild Mono runtime for all configured architectures regardless
+    # of whether a cached copy was used.
+    rebuild-mono
+
+	# Rebuild and install Mono runtime for the armeabi architecture only regardless
```

Comment:
nit: indentation.

This says ` Rebuild **and install** Mono` while the above only says Rebuild, is that intentional i.e. does it just build and not install? Same in the rebuild-help target.